src/iuse_actor.cpp

```diff
@@ -2193,6 +2193,7 @@ int fireweapon_off_actor::use( player &p, item &it, bool t, const tripoint & ) c
     if( rng( 0, 10 ) - it.damage_level( 4 ) > success_chance && !p.is_underwater() ) {
         if( noise > 0 ) {
             sounds::sound( p.pos(), noise, sounds::sound_t::combat, _( success_message ) );
+            p.add_msg_if_player( _( success_message ) );
         } else {
             p.add_msg_if_player( _( success_message ) );
         }
```

Comment:
```suggestion
        if( noise > 0 ) {
            sounds::sound( p.pos(), noise, sounds::sound_t::combat, _( success_message ) );
        }
        p.add_msg_if_player( _( success_message ) );
```
as `add_msg_if_player` is called regardless of condition, it could be simplified.